hack/third_party/gimme/gimme

```diff
@@ -285,7 +285,7 @@ _extract() {
 
 # _setup_bootstrap
 _setup_bootstrap() {
-	local versions=("1.14" "1.13" "1.12" "1.11" "1.10" "1.9" "1.8" "1.7" "1.6" "1.5" "1.4")
+	local versions=("1.18" "1.17" "1.16" "1.15" "1.14" "1.13" "1.12" "1.11" "1.10" "1.9" "1.8" "1.7" "1.6" "1.5" "1.4")
```

Comment:
1.19?

Reply:
this is directly from upstream, unmodified, and it doesn't actually have to know about 1.19, because we're already using 1.19 using this script :-)
> • upgrade gimmee